### PR TITLE
CTP-1581 : fixed remaining PHPUnit test that was failing with Moodle 4.0+

### PR DESCRIPTION
**Pull request by opitz**
CTP-1688 : removed SAGE evaluation option and use of XML-RPC
    
    removed Travis test file
    updated Behat test scenarios to match new Question Book layout from MOODLE_401_STABLE on
    now running GHA tests against MOODLE_400_STABLE, MOODLE_401_STABLE and master

### CODE REVIEW

**Review by aspark21 — COMMENTED** [review]
Overall looks good to me.

Noting there are some lines commented out in behat feature file changes, do those need to be there?

Think this will also need an update to version.php version, requires & release at a minimum

**Comment by aspark21** [comment]
P.S. Will also need to update the build (https://github.com/ucl-isd/moodle-build/tree/MOODLE_311_STABLE) for prod to look at MOODLE_311_STABLE branch for this plugin before merging.

Is this version meant to also compatible with 3.11?
github actions & separate branches would imply no, but most of these coding standards changes are not version specific.

**Comment by opitz** [comment]
This version will not cover 3.11 so we need a separate MOODLE_311_STABLE branch. I will re-version this as well next.

**Review by aydevworks — COMMENTED** [review]
It looks all good to me.

File: version.php [hunk 1]
```diff
@@ -18,15 +18,15 @@
  * The version file for qtype_algebra
  *
  * @package    qtype_algebra
- * @copyright  Roger Moore <rwmoore@ualberta.ca>
```
Comment (aspark21) [inline version.php:19]: seems a bit harsh to remove them, can't we list multiple contributors?

Comment (opitz) [reply version.php:19]: have amended now.

**Comment by aspark21** [comment]
Final requirement for merging this is the change to moodle-build I mentioned earlier.

This is now good to merge but doing so will break the Moodle 3.11 prod build so need that addressed beforehand.
Just reviewed the MR - https://github.com/ucl-isd/moodle-build/pull/10 - which requires some changes, once, that's in, this can be merged.

**Comment by opitz** [comment]
I have changed the build.sh accordingly,  pushed this to branch CTP-1581  and asked for a merge here:
https://github.com/ucl-isd/moodle-build/pull/10

Wrong procedure?

**Review by aspark21 — APPROVED** [review]

**Pull request merged**